exporter/awskinesisexporter/exporter.go

```diff
@@ -86,6 +86,10 @@ func createExporter(c config.Exporter, log *zap.Logger) (*Exporter, error) {
 		return nil, err
 	}
 
+	if conf.Encoding.Name == "otlp_json" {
+		log.Info("otlp_json is considered experimental and should not be used in a production environment")
```

Comment:
Have seen this is experimental still in 2024, is this still true? If it is could you provide some guidance to why